packages/entity-list/src/main.js

```diff
@@ -69,7 +81,11 @@ EntityListApp.propTypes = {
   showSearchForm: React.PropTypes.bool,
   preselectedSearchFields: React.PropTypes.object,
   disableSimpleSearch: React.PropTypes.bool,
-  simpleSearchFields: React.PropTypes.arrayOf(React.PropTypes.string)
+  simpleSearchFields: React.PropTypes.arrayOf(React.PropTypes.string),
+  ...EXTERNAL_EVENTS.reduce((propTypes, event) => {
```

Comment:
griffelspitz : ...EXTERNAL_EVENTS.reduce((propTypes, event) => ({...propTypes, [event]: React.PropTypes.func}), {}) should work as well